### DISCUSSION TITLE
Publish dynamic cli as a pip package

### DISCUSSION THREAD

**Discussion by lainq** [IndianOpenSourceFoundation/dynamic-cli#131] [General]
I think it would be better for people to test out the project if they could just install it quickly. I think we can deploy dynamic cli as a pip package so that people can download and test it quickly. Please let me know what you guys think about this!

**Comment by GouravSardana** [comment]
I'm waiting for the first MVP which includes the autocomplete part. If that will be done, we are good to go and will make a website for dynamic as well.
